jbmc/src/java_bytecode/ci_lazy_methods.cpp

```diff
@@ -382,6 +382,8 @@ void ci_lazy_methodst::initialize_instantiated_classes(
       if(param.type().id()==ID_pointer)
       {
         const pointer_typet &original_pointer=to_pointer_type(param.type());
+        add_clinit_call_for_pointer_type(
```

Comment:
:no_entry_sign: I can't remember exactly where it is, but somewhere we try using the `pointer_type_selector`, you should probably `add_clinit_call_for_pointer_type` for the replacement pointer too.

Reply:
Nope - this is fine because `add_all_needed_classes` calls `initialize_instantiated_classes` with both the original and any substituted pointer